lib/kitchen/driver/vagrant.rb

```diff
@@ -135,6 +135,20 @@ def destroy(state)
         state.delete(:hostname)
       end
 
+      def package(state)
+        if state[:hostname].nil?
+          raise UserError, "Vagrant instance not created!"
+        end
+        if not (config[:ssh] && config[:ssh][:insert_key] == false)
+          m = "Disable vagrant ssh key replacement to preserve the default key!"
+          warn(m)
+        end
+        instance.transport.connection(state).close
+        box_name = File.join(Dir.pwd, instance.name + ".box")
+        run("#{config[:vagrant_binary]} package --output #{box_name}")
+        destroy(state)
```

Comment:
The package command stops the machine, and I found testkitchen had trouble communicating with it if I re-ran vagrant up (ssh would keep failing). I'm not sure why.